tests/aws/services/s3/test_s3.py

```diff
@@ -6700,28 +6699,19 @@ def test_s3_copy_md5(self, s3_bucket, snapshot, monkeypatch, aws_client):
         assert request_response.status_code == 200
 
     @markers.aws.only_localstack
-    @pytest.mark.parametrize("case_sensitive_headers", [True, False])
-    def test_s3_get_response_case_sensitive_headers(
-        self, s3_bucket, case_sensitive_headers, aws_client
-    ):
-        # Test that RETURN_CASE_SENSITIVE_HEADERS is respected
+    def test_s3_get_response_case_sensitive_headers(self, s3_bucket, aws_client):
+        # Test that ETag headers is canse sensitive
```

Comment:
thanks for the fix to the test! I think it makes sense this way
```suggestion
    def test_s3_get_response_case_sensitive_headers(self, s3_bucket, aws_client):
        # Test that ETag headers is case sensitive
```